### ISSUE TITLE
[Tabs] Improve UX of overflow affordance

### ISSUE THREAD

**Issue by macandcheese** [Esri/calcite-design-system#10177]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Description

Currently when the Tab Titles overflow horizontally, it is very easy to accidentally dismiss a `closable` Tab Title in the last position while moving through Tab Titles via the pagination arrows.

Additionally, from #10324, when you have multiple tabs open, the close button for the last tab can become hidden behind the navigation arrow. When this occurs there is no way to close the last tab in the list.

https://github.com/user-attachments/assets/cc6d4540-d2a9-4a38-b2fa-c6627e03592b

### Acceptance Criteria

I would expect the pagination arrow affordances to persist, and perhaps have a disabled state when the area is at an extent. Many examples in the wild have these "together" at the beginning or end, and this could be something to explore here as well.

[Figma file](https://www.figma.com/design/XsRCqLTnKtczZr0sMEeq18/Tabs---Improve-UX-of-overflow-affordance?node-id=301-305&m=dev&focus-id=1502-2690)

Before/after example:
![CleanShot 2024-12-09 at 10 04 00@2x](https://github.com/user-attachments/assets/85d30d8d-221a-4322-8723-53ff3693425c)

### Relevant Info

Related/duplicate issue: #10324

### Which Component

Tabs

### Example Use Case

As a user, it is frustrating and destructive to accidentally dismiss a Tab Title I did not intend to, as I am using the pagination arrows to navigate the overflowed Titles.

### Priority impact

impact - p1 - need for current milestone

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

Calcite (design)

**monday.com sync:** #9366207210

**Comment by geospatialem** [comment]
The work above is dependent on https://github.com/Esri/calcite-design-system/pull/10532 landing.

> ### Acceptance Criteria
> I would expect the pagination arrow affordances to persist, and perhaps have a disabled state when the area is at an extent. Many examples in the wild have these "together" at the beginning or end, and this could be something to explore here as well.

@ashetland @SkyeSeitz For the overflow affordance in the "Acceptance Criteria" above, is the intent to have the affordance persist only at the end of the component? cc @macandcheese

**Comment by ashetland** [comment]
That's correct. I added a before/after example to the issue description for clarity.

**Comment by macandcheese** [comment]
@ashetland is there an example of the default appearance type, not bordered?

**Comment by ashetland** [comment]
Updated the image in the description to include that and added the Figma link.

**Comment by ashetland** [comment]
> Blocking based on [#10177 (comment)](https://github.com/Esri/calcite-design-system/issues/10177#issuecomment-2441662100).

I'm not sure this needs to be blocked. What requires component tokens for the above design change?

**Comment by DintaMel** [comment]
https://github.com/user-attachments/assets/da20d3de-a918-4436-95f0-39d81df27de2
Verified with 4.0.0-next.33 via [CodePen URL](https://codepen.io/DintaMel-the-lessful/pen/jEWpNzo)
